packages/notebook/test/actions.spec.ts

```diff
@@ -1111,6 +1111,24 @@ describe('@jupyterlab/notebook', () => {
         expect(result).toBe(true);
         expect(widget.isSelected(widget.widgets[2])).toBe(true);
       });
+
+      it('should run the selected cells', async () => {
+        let emitted = 0;
+        const next = widget.widgets[2];
+        NotebookActions.selectionExecuted.connect(async (_, args) => {
+          const { notebook, lastCell } = args;
+          expect(notebook).not.toBe(null);
+          expect(lastCell).not.toBe(null);
```

Comment:
This works, but it would be a better practice to define `notebook` and `lastCell` one level higher and just assign it here from `args`, and the move the `expect` to the main body of the test.